port/win/port_win.cc

```diff
@@ -101,8 +101,12 @@ bool CondVar::TimedWait(uint64_t abs_time_us) {
   std::unique_lock<std::mutex> lk(mu_->getLock(), std::adopt_lock);
 
   // Work around https://github.com/microsoft/STL/issues/369
-#if defined(_MSC_VER) && \
-    (!defined(_MSVC_STL_UPDATE) || _MSVC_STL_UPDATE < 202008L)
+  // std::condition_variable_any::wait_for had a fix,
+  // use the following two lines of directives instead
+  // #if defined(_MSC_VER) && \
+  //     (!defined(_MSVC_STL_UPDATE) || _MSVC_STL_UPDATE < 202008L)
```

Comment:
Sorry I don't know when someone should use these two lines of directives. If they're not needed can we delete them?

Reply:
Removed